docs/read_the_docs/source/advanced_tutorial/openstudio_hpxml.rst

```diff
@@ -0,0 +1,43 @@
+OpenStudio-HPXML
```

Comment:
Add a new "OpenStudio-HPXML" section to the Advanced Tutorial? Currently it would have two subsections:
* syncing (this is updating resstock's develop branch to match OS-HPXML's master branch)
* testing (this is using resstock to test an umerged branch of OS-HPXML)